### ISSUE TITLE
Privacy Badger injects Facebook like buttons in articles on thuglifer.com

### ISSUE THREAD

**Issue by anonsubmitter** [EFForg/privacybadgerfirefox#687]
Privacy Badger seems to inject Facebook like buttons in the bottom of articles on thuglifer.com.
Sample: http://thuglifer.com/guy-steals-a-tank-and-causes-destruction-around-san-diego/

**Comment by Calefornia** [comment]
That is an intended feature, and it can be changed in your settings.... I believe requests are still blocked, but you will no longer see the FB buttons.
Other Settings
[ ] Social widget replacement enabled

**Comment by cooperq** [comment]
correct, it is not injecting the facebook button but replacing the one that is already there with one that wont track you. This behavior can be disabled if you don't like it.

**Comment by anonsubmitter** [comment]
I know about that function. What is weird is that there is no Facebook like button there originally, Privacu Badger injects it. I looked at the page in an out of the box Pale Moon installation and there is no Facebook like button there before Privacy Badger is used.

**Comment by Calefornia** [comment]
Does Pale Moon offer some built-in privacy features or tracking protection, like Firefox? I disabled privacy badger and loaded the site, and I saw what you meant: there was no FB button. However, when I disabled the Firefox protection, the FB buttons were loaded. They do exist, but these other features are to blame.... 
![screen shot 2015-12-08 at 10 03 41 am](https://cloud.githubusercontent.com/assets/7418271/11663815/321b3d44-9d93-11e5-91d0-b57a58da1880.png)
